vault/resource_identity_oidc_provider.go

```diff
@@ -27,8 +28,60 @@ func identityOIDCProviderResource() *schema.Resource {
 			"issuer": {
 				Type: schema.TypeString,
 				Description: "Specifies what will be used as the 'scheme://host:port' component for the 'iss' claim of ID tokens." +
-					"If provided explicitly, it must point to a Vault instance that is network reachable by clients for ID token validation.",
+					"This value is computed using the issuer_host and https_enabled schema fields.",
 				Computed: true,
+				// TODO confirm if this is needed
+				Optional: true,
```

Comment:
@benashz I had to mark this field as `Optional` to make the `DiffSuppressFunc` usable, though I'm not sure if either marking it as Optional or the Diff Suppression is still necessary